consensus/istanbul/core/core.go

```diff
@@ -422,3 +422,17 @@ func PrepareCommittedSeal(hash common.Hash) []byte {
 	buf.Write([]byte{byte(msgCommit)})
 	return buf.Bytes()
 }
+
+// Minimum required number of consensus messages to proceed
+func requiredMessageCount(valSet istanbul.ValidatorSet) int {
+	size := valSet.Size()
+	switch size {
+	// in the certain cases we must receive the messages from all consensus nodes to ensure finality...
+	case 1, 2, 3:
+		return int(size)
+	case 6:
+		return 5 // when the number of valSet is 6 and return value is 2*F+1, the return value is not safe. It should return 4 or more.
```

Comment:
just a typo: It should return 5 or more.

Reply:
Thanks for your review.
I changed the return value from 5 to 4.
It's reflected in https://github.com/klaytn/klaytn/pull/1462/commits/f48998be7bbacd71a084660d8be4f918a87cda99.